cmd/horcrux/cmd/testdata/config-migrated.yaml

```diff
@@ -1,8 +1,8 @@
 cosigner:
   threshold: 2
-  shares: 3
-  p2p-listen: tcp://127.0.0.1:2222
   peers:
+  - share-id: 1
+    p2p-addr: tcp://127.0.0.1:2222
```

Comment:
from our discussion off GH, we talked about having these be `shards` instead of `peers`, with an `id` property within each. After playing around with it some more I changed it to `cosigners` instead of `peers`, and changed `share-id` -> `shardID` within. I changed the higher level `cosigner` property to `thresholdMode`, since I think it better represents the configuration that is specific to using `threshold` mode. The grouping of `shardID` and `p2pAddr` adequately represents what cosigners need to know about each other, the network reachable address and the key shard ID for the ephemeral shares (nonce) and combined signature.